README.md

```diff
@@ -110,6 +110,32 @@ import bqplot
 
 That's it! You're ready to go!
 
+### Running tests
```

Comment:
Thanks for adding docs, if this is missing, please add it to the `CONTRIBUTING.md` file instead of the README

Reply:
k!

Reply:
Actually, I was quite wrong. It's not `CONTRIBUTING.md` where the dev installation instructions are, it's actually in the README but it's earlier in the file.